packages/amplication-client/src/VersionControl/PendingChanges.scss

```diff
@@ -8,6 +8,8 @@ $counter-height: 18px;
   display: flex;
   flex-direction: column;
   flex: 1;
+  overflow-y: auto;
+  overflow-x: hidden;
```

Comment:
Do we need this?

Reply:
Yes, its sticks the view code in the bottom
<img width="1134" alt="Screen Shot 2022-08-21 at 19 46 44" src="https://user-images.githubusercontent.com/61761153/185801920-0174d165-59f5-4a62-a673-08176fed5016.png">